toxcore/crypto_core.c

```diff
@@ -142,20 +141,23 @@ int32_t encrypt_data_symmetric(const uint8_t *secret_key, const uint8_t *nonce,
         return -1;
     }
 
-    VLA(uint8_t, temp_plain, length + crypto_box_ZEROBYTES);
-    VLA(uint8_t, temp_encrypted, length + crypto_box_MACBYTES + crypto_box_BOXZEROBYTES);
+    uint8_t* temp_plain = calloc(length + crypto_box_ZEROBYTES, sizeof(uint8_t));
+    uint8_t* temp_encrypted = malloc(length + crypto_box_MACBYTES + crypto_box_BOXZEROBYTES);
```

Comment:
Please be consistent. Choose one, and do that everywhere (probably calloc?).

Reply:
Not relevant any more. This place was fixed by zoff99.